mods/ra/maps/intervention/rules.yaml

```diff
@@ -76,6 +76,7 @@ MIG:
 		Prerequisites: ~afld
 	AmmoPool:
 		Ammo: 2
+	-Rearmable:
```

Comment:
The unit is still buildable, so not sure if that is a good idea. If we have to do such changes we'll need to clone the unit. (Btw, do you think the `LeaveMap` `IdleBehavior` can help here?)

Reply:
I like the IdleBehavior suggestion. I lost track of that and wasn't aware that it had already been merged.